libcontainer/configs/config_test.go

```diff
@@ -153,7 +155,35 @@ func TestFuncHookRun(t *testing.T) {
 		return nil
 	})
 
-	fHook.Run(state, nil)
+	if err := fHook.Run(state); err != nil {
+		t.Fatal(err)
+	}
+}
+
+func testCommandHookRun(t *testing.T, state interface{}, command string) {
+	tmpFile, err := ioutil.TempFile("", "runcTest")
+	if err != nil {
+		t.Fatalf("Failed to create tmp file: %v", err)
+	}
+	defer os.Remove(tmpFile.Name())
+	if _, err := tmpFile.Write([]byte(command)); err != nil {
+		t.Fatalf("Failed to write tmp file: %v", err)
+	}
+	if err := tmpFile.Chmod(0777); err != nil {
```

Comment:
```suggestion
	if err := tmpFile.Chmod(0700); err != nil {
```

That should be enough?